yelp_package/Makefile

```diff
@@ -73,7 +73,6 @@ package_%: build_$$*_docker go_entrypoint
 
 	# noddebs to work around https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=897569
 	$(DOCKER_RUN) /bin/bash -c "DEB_BUILD_OPTIONS=noddebs dpkg-buildpackage -d && mv ../*.deb dist/"
-	$(DOCKER_RUN) chown -R $(UID):$(GID) /work
```

Comment:
This will still be needed in case this is built with actual docker, so I'd just suggest wrapping it in
```
ifeq ($(findstring podman,$(DOCKER_HOST)),)
...
endif
```